sdk/nodejs/policy/policy.ts

```diff
@@ -82,10 +69,142 @@ export interface Policy {
      * proper permissions.
      */
     enforcementLevel: EnforcementLevel;
+}
+
+/**
+ * An array of Policies.
+ */
+export type Policies = (ResourceValidationPolicy | StackValidationPolicy)[];
```

Comment:
Question bc Im still a typescript noob. Does this mean you can have an array of all `ResourceValidationPolicy` type or all `StackValidationPolicy `, but not a mix of both? I think we want to be able to define both in the same policy pack. Just want to confirm, bc this has tripped me up in the past.

Reply:
Mix of both. I created a [Playground](http://www.typescriptlang.org/play/?ssl=47&ssc=1&pln=7&pc=1#code/KYDwDg9gTgLgBASwHY2FAZgQwMbDgBQgBsFsBPfHAawEEoBzAZzgG8AoOTuSE7BYRgC4CxUv0YBuNgF82bUJFhxsRTI2aFeFaqw5cwUBADdMqOFGCYAJhCREy3UXwHDNYgVL2dstxjCgArtgw0AAUBsameEiYALbAwn6GSPQANHCYDEIiWpTYtFkAlLpcpXAwABYIjAB0PO7MALwZWXVO4lKlsrLy4NDwMGRgeG7OTXChAEoCEAFQuABqmCRWpgi2ow4APnAAyjDUSytrG05khQDaALqeCv3lQ3gAokjo0LjxKAAywEbARHBmgAiaxGarQMhAuA7IGxTBIVYhKCQ259JTIVAYHAjM4lLgxeKJfzIeidLhWATYQxgGDrJBE5KkrxwYCvd7AT4wH5-IjCF5veYc1lc37-KQ9O7olBoLC4ODTRizQVHBCIumbFkgVAIjS49ilEzHVAKpW4YQAIwgxEsSAkcAA9Pa4ALEEQiAEkms-tw5pBGAIZHJJfAMTLsXsDvkVWrTlpNdqrLq4-quIbVVF9tQLVaiDa7Y7ndBXe7PbTvWBfRB-YxA2wfEg-I5eOJAXAkMAAO45UjafKhIFgMBA9Ipzj1MbCC7M0qjsqlAkJOBAiyKua4AC0aZjSHX48hqWnc7gFMYVIQNLpwiBJ+ww8Pc9ZAo+wu5-yvoPByLvR7KW6iJrXRd-ACYB7zgaQDx-WcfwXK8-GoTdlnTWlbF3M5vx-clKWpFD6SXG8MMwlk2UFTlX15JcP0VL9IKIuA-1QTN8mEYDQJ-CDmSuA9pEKKQgA) that demonstrates.

If it was all `ResourceValidationPolicy` or all `StackValidationPolicy` (which we don't want), it'd be typed as `ResourceValidationPolicy[] | StackValidationPolicy[]`.